<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPASS_DISTRIBUTION</class>
 <widget class="QDialog" name="XPASS_DISTRIBUTION">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="GB_AREA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>132</height>
    </rect>
   </property>
   <property name="title">
    <string>Rectangle area</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="LP0">
        <property name="text">
         <string>point0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LEP0"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="LP1">
        <property name="text">
         <string>point1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LEP1"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LP2">
        <property name="text">
         <string>point2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LEP2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="LP3">
        <property name="text">
         <string>point3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="LEP3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="PROB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>85</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="CB_TXT">
      <property name="text">
       <string>Export text</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_SelectPart">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Select part</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ANALYSIS">
      <property name="minimumSize">
       <size>
        <width>83</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>83</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Analysis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Exit">
      <property name="minimumSize">
       <size>
        <width>83</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>83</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
